File: Java/Beginner/1008/Main.java
import java.util.Scanner;

class Main {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);

            int num = scan.nextInt(), hours = scan.nextInt();
            double salary = scan.nextDouble(), total;
            total = salary * hours;

            System.out.printf("NUMBER = %d\nSALARY = U$%.2f\n", num, total); 

        scan.close();
    }

}

File: C/Beginner/1008/main.c
#include <stdio.h>
 
int main() {
 
    int number, hourTotal;
    double hourCost, salary;
    
    scanf("%d%d%lf", &number, &hourTotal, &hourCost);
    
    salary = hourTotal * hourCost;
    
    printf("NUMBER = %d\nSALARY = U$ %.2lf\n", number, salary);
 
    return 0;
}